File: ai-form-assistant-node/src/modules/users/controller.js
const userService = require("./service");

class UserController {

    async getUsers(req, res) {
        const users = await userService.getUsers();
        res.json(users);
    }

    async getUser(req, res) {
        const user = await userService.getUser(req.params.id);
        if (!user) {
            return res.status(404).json({ error: "User not found" });
        }
        res.json(user);
    }

}

module.exports = new UserController();

File: ai-form-assistant-fastapi/app/modules/users/controller.py
from sqlalchemy.orm import Session
import uuid

from .service import user_service


class UserController:

    def get_users(self, db: Session):
        return user_service.get_users(db)

    def get_user(self, db: Session, user_id: uuid.UUID):
        return user_service.get_user(db, user_id)


user_controller = UserController()